Add Tokenomics page (#254)
graphprotocol/docs: pages/en/tokenomics.mdx

# The Roles of Network Participants

There are four primary network participants:

1.  Delegators - Delegate GRT to Indexers & secure the network

2.  Curators - Find the best subgraphs for Indexers

3.  Developers - Build & query subgraphs

4.  Indexers - Backbone of blockchain data

Fishermen and Arbitrators are also integral to the network's success through other contributions, supporting the work of the other four. For more information about network roles, [read this article](https://thegraph.com/blog/the-graph-grt-token-economics/).

![Tokenomics diagram](/img/Tokeneconomics-Diagram@2x.png)

# Delegators (Passively earn GRT)

Delegators delegate their GRT to Indexers, which helps Indexers increase their stake in subgraphs on the network. In return, Delegators earn a percentage of all query fees and indexing rewards from the Indexer. Each Indexer sets the cut that will be rewarded to Delegators independently, creating competition among Indexers to attract Delegators. Most Indexers offer between 9-12% annually.

For example, if a Delegator were to delegate 15k GRT to an Indexer offering 10%, the Delegator would receive ~1500 GRT in rewards annually.

> There is a 0.5% delegation tax which is burned whenever a Delegator delegates GRT on the network. If a Delegator chooses to withdraw their delegated GRT, the Delegator must wait for the 28 epoch unbonding period. Each epoch is 6,646 blocks, which means 28 epochs ends up being approximately 26 days.

Reviewer: ```suggestion
There is a 0.5% delegation tax which is burned whenever a Delegator delegates GRT on the network. If a Delegator chooses to withdraw their delegated GRT, the Delegator must wait for the 28-epoch unbonding period. Each epoch is 6,646 blocks, which means 28 epochs ends up being approximately 26 days.
```

If you're reading this, you're capable of becoming a Delegator right now by heading to the [network participants page](https://thegraph.com/explorer/participants/indexers), and delegating GRT to an Indexer of your choice.

# Curators (Earn GRT)

> Curators identify high-quality subgraphs, and "curate" them (i.e., signaling GRT on them) to earn curation shares, which guarantee a percentage of all future query fees generated by the subgraph. While any independent network participant can be a Curator, typically subgraph developers are among the first Curators for their own subgraphs because they want to ensure their subgraph is indexed.

Reviewer: ```suggestion
Curators identify high-quality subgraphs, and "curate" them (i.e., signal GRT on them) to earn curation shares, which guarantee a percentage of all future query fees generated by the subgraph. While any independent network participant can be a Curator, typically subgraph developers are among the first Curators for their own subgraphs because they want to ensure their subgraph is indexed.
```

As of December 2022, subgraph developers are encouraged to curate their subgraph with at least 10,000 GRT. However, this number may be impacted by network activity and community participation.

Curators pay a 1% curation tax when they curate a new subgraph. This curation tax is burned, decreasing the supply of GRT.

# Developers

> Developers build and query subgraphs to retrieve blockchain data. Since subgraphs are open-source, developers can query existing subgraphs to load blockchain data into their dapps. Developers pay for queries they make in GRT, that is distributed to network participants.

Reviewer: ```suggestion
Developers build and query subgraphs to retrieve blockchain data. Since subgraphs are open source, developers can query existing subgraphs to load blockchain data into their dapps. Developers pay for queries they make in GRT, that is distributed to network participants.
```

## Creating a subgraph

> Developers can [create a subgraph](https://thegraph.com/docs/en/developing/creating-a-subgraph/) to index data on the blockchain. Subgraphs are instructions for Indexers about which data should be served to consumers.

Reviewer: ```suggestion
Developers can [create a subgraph](/developing/creating-a-subgraph/) to index data on the blockchain. Subgraphs are instructions for Indexers about which data should be served to consumers.
```
Relative links will ensure that the language is preserved without having to translate the URL in Crowdin. It's also more future proof, in case we change the base URL to docs.thegraph.com or something (though we'd certainly have redirects in place, so it's mostly for the first reason, and because it's shorter :P).

Author: Thanks for catching that! Always forget whenever we go from Google Docs to markdown.

> Once developers have built and tested their subgraph, they can [publish their subgraph](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) on The Graph's decentralized network.

Reviewer: ```suggestion
Once developers have built and tested their subgraph, they can [publish their subgraph](/publishing/publishing-a-subgraph/) on The Graph's decentralized network.
```

## Querying an existing subgraph

> Once a subgraph is [published](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) to The Graph's decentralized network, anyone can create an API key, fill it with GRT, and query it.

Reviewer: ```suggestion
Once a subgraph is [published](/publishing/publishing-a-subgraph/) to The Graph's decentralized network, anyone can create an API key, fill it with GRT, and query it.
```

Reviewer: Also, fill what with GRT? The API key? The subgraph? I'm not sure about the wording here.

Author: Rephrased to be more specific - good call!

> Subgraphs are [queried using GraphQL](https://thegraph.com/docs/en/querying/querying-the-graph/), and the query fees are paid for with GRT in [Subgraph Studio](https://thegraph.com/studio/). Query fees are distributed to network participants based on their contributions to the protocol.

Reviewer: ```suggestion
Subgraphs are [queried using GraphQL](/querying/querying-the-graph/), and the query fees are paid for with GRT in [Subgraph Studio](https://thegraph.com/studio/). Query fees are distributed to network participants based on their contributions to the protocol.
```

1% of the query fees paid to the network are burned.

# Indexers (Earn GRT)

Indexers are the backbone of The Graph. They operate independent hardware and software to power the network. Indexers serve data to consumers based on instructions from subgraphs.

Indexers can earn GRT rewards in two ways:

1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are deposited into a rebate pool and distributed to Indexers.

2.  Indexing rewards: the 3% annual issuance is distributed to Indexers based on the number of subgraphs they are indexing. Indexing rewards incentivize Indexers to index subgraphs, sometimes before the query fees begin to accrue, and to submit Proofs of Indexing (POIs), which verify that they have indexed data accurately.

> Each subgraph is allotted a portion of the total network token issuance, based on the amount of curation signal that subgraph has. That amount is then divided between all the Indexers staked on that subgraph based on their allocated stake.

Reviewer: "divided between all the Indexers staked" reads a bit funny to me.

Author: Reworded it! Good call!

In order to run an indexing node, Indexers must stake 100,000 GRT or more with the network. Indexers are incentivized to stake GRT in proportion to the amount of queries they serve.

> Indexers can increase their GRT allocations on subgraphs by accepting GRT delegation from Delegators, and they can accept up to 16X GRT of their initial. If an Indexer becomes "over-delegated" (i.e., more than 16X their initial stake), they will not be able to use the additional GRT from Delegators until they increase their stake in the network.

Reviewer: ```suggestion
Indexers can increase their GRT allocations on subgraphs by accepting GRT delegation from Delegators, and they can accept up to 16 times their initial stake. If an Indexer becomes "over-delegated" (i.e., more than 16 times their initial stake), they will not be able to use the additional GRT from Delegators until they increase their stake in the network.
```

The amount of rewards an Indexer receives can vary based on the initial stake, accepted delegation, quality of service, and many more factors. The following chart is publicly available data from an active Indexer on The Graph's decentralized network.

## The Indexer stake & reward of allnodes-com.eth

> ![Indexing stake and rewards](/img/indexing-stake-and-income.png) This data is from February 2021 to September 2022.

Reviewer: ```suggestion
![Indexing stake and rewards](/img/indexing-stake-and-income.png)

This data is from February 2021 to September 2022.
```

Doesn't change much (if anything) visually because images are forced to be full width at the moment, so the text wraps to the next line but it is technically inline with the image in your version. This puts it on the next paragraph, which I feel is more correct.

> Please note, this will improve when the [Arbitrum migration](https://forum.thegraph.com/t/gip-0037-the-graph-arbitrum-deployment-with-linear-rewards-minted-in-l2/3551) is complete, making gas costs a significantly lower burden for participating on the network.

# Token Supply: Burning & Issuance

The initial token supply is 10 billion GRT, with a target of 3% new issuance annually to reward Indexers for allocating stake on subgraphs. This means that the total supply of GRT tokens will increase by 3% each year as new tokens are issued to Indexers for their contribution to the network.

The Graph is designed with multiple burning mechanisms to offset new token issuance. Approximately 1% of the GRT supply is burned annually through various activities on the network, and this number has been increasing as network activity continues to grow. These burning activities include a 0.5% delegation tax whenever a Delegator delegates GRT to an Indexer, a 1% curation tax when Curators signal on a subgraph, and a 1% of query fees for blockchain data.

> In addition to these regularly occurring burning activities, the GRT token also has a slashing mechanism in place to penalize malicious or irresponsible behavior by Indexers. If an Indexer is slashed, 50% of their indexing rewards for the epoch are burned (while the other half goes to the fisherman), and their self-stake will be slashed by 2.5%, with half of this amount being burned. This helps to ensure that Indexers have a strong incentive to act in the best interests of the network and to contribute to its security and stability.

Reviewer: ```suggestion
In addition to these regularly occurring burning activities, the GRT token also has a slashing mechanism in place to penalize malicious or irresponsible behavior by Indexers. If an Indexer is slashed, 50% of their indexing rewards for the epoch are burned (while the other half goes to the fisherman), and their self-stake is slashed by 2.5%, with half of this amount being burned. This helps to ensure that Indexers have a strong incentive to act in the best interests of the network and to contribute to its security and stability.
```

# Improving the Protocol

> The Graph Network is ever-evolving and improvements to the economic design of the protocol are constantly being made to provide the best experience for all network participants. The Graph Council oversees protocol changes and community members are encouraged to participate.\
> Get involved with protocol improvements in [The Graph Forum](https://forum.thegraph.com/).

Reviewer: Did you mean to include a line break here? If not:

```suggestion
The Graph Network is ever-evolving and improvements to the economic design of the protocol are constantly being made to provide the best experience for all network participants. The Graph Council oversees protocol changes and community members are encouraged to participate. Get involved with protocol improvements in [The Graph Forum](https://forum.thegraph.com/).
```

---
title: Tokenomics of The Graph Network
navTitle: Tokenomics

> description: The Graph Network is incentivized by powerful tokenomics. Here’s how GRT, The Graph’s native work utility token works.

Reviewer: Just noticed this. We don't do anything with a field called `description` at the moment; what are you trying to do?

Author: This was an SEO description/subtitle. Any other way to include this for SEO purposes?

Reviewer: Handled in https://github.com/graphprotocol/docs/pull/254/commits/a556f8d2e2e6a5c9ed41503e21921a65b0728bd7
